src/tutorial/microbe_editor/EditorUndoTutorial.cs

```diff
@@ -32,9 +32,13 @@ public override bool CheckEvent(TutorialState overallState, TutorialEventType ev
         {
             case TutorialEventType.MicrobeEditorOrganellePlaced:
             {
-                if (!HasBeenShown && CanTrigger && !overallState.TutorialActive())
+                OrganelleDefinition placedOrganelle = ((OrganellePlacedEventArgs)args).Definition;
+
+                if (!HasBeenShown && CanTrigger && !overallState.TutorialActive()
```

Comment:
I really think we need to tutorialize the undo feature earlier than when placing the nucleus.

We could have another tutorial reminding about undo when placing the nucleus (if the player ATP balance / resource balance is not good). But I don't think we should give up the undo tutorial, because back in the dark ages people used to often ask on our discord how to undo. And I don't want to go back to that.

Reply:
This isn't the undo tutorial waiting until the player is ready for the nucleus, this is the editor tutorial telling the player to put a nucleus on _first_, then explaining why that would be a terrible idea and explaining the undo button.

It is entirely possible for a player to ignore this advice and skip this phase by doing different modifications, but I opted to not force the player to read more walls of text if they are aren't ignoring instructions.

Reply:
But then the player is allowed to skip like half of the hands on tutorial for the first editor cycle, which I don't think is good...

Reply:
The player is allowed to ship all of the phases by closing them (or just not paying attention). Hopefully reducing four consecutive text boxes to just three will let us hold on to that attention longer for the more important information.

Plus, this isn't canceling the other editor tutorial buttons; two boxes (showing non-vital information) are just not shown until a player chooses to try the nucleus.

Reply:
Is there safety against the player just placing another kind of organelle and then confirming to exit the editor? That would be really bad if the player can skip the most important tutorial in the game like that. I think that the changes here make it much more likely because if the undo tutorial doesn't trigger for every placed organelle and the nucleus needs to be found by scrolling down the list of available organelles, I think we are in a similar situation as the glucose balance display which is not visible by default.